src/components/NavBar /NavBar.jsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Box,
+  Button,
+  Drawer,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  IconButton,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import { BiBookBookmark as ShoppingCartIcon } from "react-icons/bi";
+import SectionSeperator from "./SectionSeperator";
+import ShoppingCartItem from "./ShoppingCartItem";
+
+const NavBar = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [shoppingCartItems] = React.useState([
+    { planName: "Ladder T-walls", imageNum: 2, commentNum: 8 },
+    { planName: "Ladder T-walls", imageNum: 3, commentNum: 6 },
+    { planName: "Ladder T-walls", imageNum: 1, commentNum: 2 },
+    { planName: "Ladder T-walls", imageNum: 4, commentNum: 5 },
+    { planName: "Ladder T-walls", imageNum: 6, commentNum: 7 },
+  ]);
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  return (
+    <div>
+      <nav>
+        <div style={NavBarContainer}>
+          <div style={NavBarElement}>Digital Library</div>
+          <div style={NavBarElement}>Project Plan Builder</div>
+          <div style={NavBarElementCart}>
+            <IconButton onClick={onOpen} icon={<ShoppingCartIcon />} />
+            <div style={NavBarElement}>Logout</div>
+          </div>
+        </div>
+      </nav>
+
+      <Drawer isOpen={isOpen} onClose={handleClose}>
+        <DrawerOverlay />
+        <DrawerContent justifyContent="space-between">
+          <Box>
+            <DrawerHeader display="flex" justifyContent="center">
+              Atlanta Construction 47
+            </DrawerHeader>
+            <SectionSeperator />
```

Comment:
Before creating any custom html components, see if it would be available in chakra ui documentation. If its not there, try your best to nest stuff in chakra. There should be no standard html components, only chakra stuff. In this case, look into using the `<Divider>` component in chakraUI